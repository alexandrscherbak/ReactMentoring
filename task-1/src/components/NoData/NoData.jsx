import React from 'react';

const NoData = () => (
	<div>
		<h1>No Films Found</h1>
	</div>
);

export default NoData;
