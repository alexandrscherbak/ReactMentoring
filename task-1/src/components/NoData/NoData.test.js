import React from 'react';
import NoData from './NoData.jsx';
import renderer from 'react-test-renderer';

test('NoData snapshot', () => {
	const component = renderer.create(
		<NoData />,
	);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
