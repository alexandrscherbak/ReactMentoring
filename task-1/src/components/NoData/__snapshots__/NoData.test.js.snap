// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoData snapshot 1`] = `
<div>
  <h1>
    No Films Found
  </h1>
</div>
`;
